pkg/webhook/mutation/pod_mutator/pod_mutator.go

```diff
@@ -24,7 +26,7 @@ const (
 
 // AddPodMutationWebhookToManager adds the Webhook server to the Manager
 func AddPodMutationWebhookToManager(mgr manager.Manager, ns string) error {
-	podName := os.Getenv(kubeobjects.EnvPodName)
+	podName := os.Getenv(env.EnvPodName)
```

Comment:
```suggestion
	podName := os.Getenv(env.PodName)
```

Reply:
done in [542e38f](https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/542e38f34e74944a9b6e7e5db9623aa2d7969e7e)